app/dashboard/_components/UploadThingModal.tsx

```diff
@@ -0,0 +1,69 @@
+'use client';
+
+import { setAppSetting } from '~/actions/appSettings';
+import { UploadThingTokenForm } from '~/app/(blobs)/(setup)/_components/UploadThingTokenForm';
+import Link from '~/components/Link';
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogHeader,
+  DialogTitle,
+} from '~/components/ui/dialog';
+import Paragraph from '~/components/ui/typography/Paragraph';
+
+function UploadThingModal() {
+  return (
+    <Dialog open={true}>
+      <DialogContent aria-describedby={undefined} className="sm:max-w-[500px]">
+        <DialogHeader>
+          <DialogTitle>Required Environment Variable Update</DialogTitle>
+          <DialogDescription>
+            <Paragraph>
+              This Fresco update requires a new UploadThing environment
+              variable. Please add it by following these steps.
+            </Paragraph>
+
+            <Paragraph>
+              <ol className="list-inside list-decimal">
+                <li>
+                  Visit the{' '}
+                  <Link
+                    href="https://uploadthing.com/dashboard/"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    UploadThing dashboard
+                  </Link>
+                </li>
+                <li>Select your project.</li>
+                <li>Select the API Keys tab.</li>
```

Comment:
I think it would be useful to have an image here, since this is a UI change on the UT side too, and there are some confusing elements to it.